.github/workflows/go.yml

```diff
@@ -34,10 +34,10 @@ jobs:
         run: if [ $(gofmt -l -s . | wc -l) -ne 0 ]; then echo "fmt failed"; exit 1; fi
 
       - name: Build
-        run: go build -v ./msal
+        run: go build -race -v ./msal
```

Comment:
All the updates look good.  But I'm wondering why we are running with -v?  In the case of test failure it dumps info, but -v just dumps pages of info we don't care about.  Both in builds and tests.

Reply:
-v was from the initial commit of the pipeline file, I'm fine dropping it